main.c

```diff
@@ -685,6 +688,16 @@ static void *tcmur_cmdproc_thread(void *arg)
 			tcmu_err("ppoll received unexpected revent: 0x%x\n", pfd.revents);
 			break;
 		}
+
+		/*
+		 * If a device being removed is dealing with contiguous I/O requests,
+		 * we need to throttle cmdproc thread to make sure it have a chance
+		 * to become idle
```

Comment:
Could you also change this comment to be:

LIO will wait for outstanding requests and prevent new ones from being sent to runner during device removal, but if the tcmu cmd_time_out has fired tcmu-runner may still be executing requests that LIO has completed. We only need to wait for replies for outstanding requests so exit from the cmdproc now.

Reply:
More precise and detailed. I will update it in next upate.